api/handlers.go

```diff
@@ -62,6 +62,59 @@ func getVersion(c *context, w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(version)
 }
 
+// GET /images/get
+func getImages(c *context, w http.ResponseWriter, r *http.Request) {
+	if err := r.ParseForm(); err != nil {
+		httpError(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	names := r.Form["names"]
+
+	// find an engine which has all images
+	// Engine.Addr : [found names]
+	dict := make(map[string][]string)
+	bFoundEngine := false
```

Comment:
I think we do not need this `bFoundEngine`.
Just check with `foundEngineAddr = ""` as Golang alreay inits it to an empty string.